recipes/typos/meta.yaml

```diff
@@ -0,0 +1,32 @@
+{% set name = "typos" %}
+{% set version = "1.13.10" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/crate-ci/typos/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: c67df6f7867e72c99187ebcad7c54bb100f4acc7a76fb939f647874b8b0c3ee4
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - rust
+
+test:
+  commands:
+    - typos --version | grep '{{ version }}'
+
+about:
+  home: https://github.com/crate-ci/typos
+  summary: 'Source code spell checker'
+  license: MIT
+  license_file: LICENSE-MIT
```

Comment:
I'm unsure about the license. It seems the repository is licensed under Apache and MIT:

- https://github.com/crate-ci/typos/blob/master/LICENSE-APACHE
- https://github.com/crate-ci/typos/blob/master/LICENSE-MIT